ovn-tester/ovn_workload.py

```diff
@@ -863,38 +388,25 @@ def ping_ports(self, ports):
         for w, ports in ports_per_worker.items():
             w.ping_ports(self, ports)
 
-    @ovn_stats.timeit
-    def provision_vips_to_load_balancers(self, backend_lists):
-        n_vips = len(self.load_balancer.vips.keys())
-        vip_ip = self.cluster_cfg.vip_subnet.ip.__add__(n_vips + 1)
-
-        vips = {
-            f'{vip_ip + i}:{DEFAULT_VIP_PORT}': [
-                f'{p.ip}:{DEFAULT_BACKEND_PORT}' for p in ports
-            ]
-            for i, ports in enumerate(backend_lists)
-        }
-        self.load_balancer.add_vips(vips)
-
-    def unprovision_vips(self):
-        if self.load_balancer:
-            self.load_balancer.clear_vips()
-            self.load_balancer.add_vips(self.cluster_cfg.static_vips)
-        if self.load_balancer6:
-            self.load_balancer6.clear_vips()
-            self.load_balancer6.add_vips(self.cluster_cfg.static_vips6)
+    def mesh_ping_ports(self, ports: List[ovn_utils.LSPort]) -> None:
```

Comment:
Is it worth accounting(`@ovn_stats.timeit`) how long the mesh_ping took in total?